Please review document title tag at the beginning

laraaleite opened the pull request:
Please review document title tag at the beginning. The text is appearing in the docs:
;﻿--- title: Monitor availability and responsiveness of any web site | Microsoft Docs description: Set up web tests in Application Insights. Get alerts if a website becomes unavailable or responds slowly. services: application-insights documentationcenter: '' author: SoubhagyaDash manager: carmonm

ms.assetid: 46dc13b4-eb2e-4142-a21c-94a156f760ee ms.service: application-insights ms.workload: tbd ms.tgt_pltfrm: ibiza ms.devlang: na ms.topic: get-started-article ms.date: 02/09/2018 ms.author: sdash ; mbullwin

azurecla commented:
Hi __@cwatsonMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (cwatson). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

squillace commented:
#sign-off

cjgronlund commented:
Will be submitted to private repo.

laraaleite committed:
Please review document title tag at the beginning

Please review document title tag at the beginning. The text is appearing in the docs:
;﻿--- title: Monitor availability and responsiveness of any web site | Microsoft Docs description: Set up web tests in Application Insights. Get alerts if a website becomes unavailable or responds slowly. services: application-insights documentationcenter: '' author: SoubhagyaDash manager: carmonm

ms.assetid: 46dc13b4-eb2e-4142-a21c-94a156f760ee ms.service: application-insights ms.workload: tbd ms.tgt_pltfrm: ibiza ms.devlang: na ms.topic: get-started-article ms.date: 02/09/2018 ms.author: sdash ; mbullwin
